Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ScrollViewerExtensions/ScrollViewerExtensionsXaml.bind

```diff
@@ -0,0 +1,114 @@
+  </Page.Resources>
+
+  <Grid>
+    <ListView Name="listView"
+              extensions:ScrollViewerExtensions.VerticalScrollBarMargin="{Binding MinHeight, ElementName=MyHeaderGrid, Converter={StaticResource DoubleTopThicknessConverter}}"
+              extensions:ScrollViewerExtensions.EnableMiddleClickScrolling="@[EnableMiddleClickScrolling:Bool:True]">
+      <ListView.Header>
+        <controls:ScrollHeader Mode="Sticky">
+          <Grid x:Name="MyHeaderGrid"
+                MinHeight="@[Header MinHeight:DoubleSlider:250:150-500]"
+                Background="{ThemeResource Brush-Sample-TransparentHeader}">
+            <StackPanel HorizontalAlignment="Center"
+                        VerticalAlignment="Center">
+              <TextBlock Margin="12"
+                         FontSize="48"
+                         FontWeight="Bold"
+                         Foreground="{ThemeResource Brush-Main}"
+                         Text="Scroll Header"
+                         TextAlignment="Center"
+                         TextWrapping="WrapWholeWords" />
+              <TextBlock Margin="12,0,12,12"
+                         VerticalAlignment="Center"
+                         Foreground="{ThemeResource Brush-Main}"
+                         Text="The attached property allows you to assign a Thickness to both vertical and horizontal ScrollBar inside any container. In this example, VerticalScrollBarMargin is bound to StickyHeader's MinHeight. Moving MinHeight's Slider will move the vertical ScrollBar accordingly."
+                         TextAlignment="Center"
+                         TextWrapping="WrapWholeWords" />
+            </StackPanel>
+          </Grid>
+        </controls:ScrollHeader>
+      </ListView.Header>
+      <ListView.ItemTemplate>
+        <DataTemplate>
+          <Grid MinHeight="100"
+                Background="{ThemeResource Brush-Grey-03}">
+            <Grid.ColumnDefinitions>
+              <ColumnDefinition Width="Auto" />
+              <ColumnDefinition Width="*" />
+            </Grid.ColumnDefinitions>
+            <Image x:Name="ToolkitLogo"
+                   Grid.Column="0"
+                   Width="100"
+                   Height="100"
+                   Margin="0,0,24,0"
+                   Source="ms-appx:///Assets/ToolkitLogo.png" />
+            <TextBlock Grid.Column="1"
+                       VerticalAlignment="Center"
+                       Text="{Binding Title}"
+                       TextTrimming="CharacterEllipsis" />
+          </Grid>
+        </DataTemplate>
+      </ListView.ItemTemplate>
+      <ListView.ItemContainerStyle>
+        <Style TargetType="ListViewItem">
+          <Setter Property="HorizontalContentAlignment" Value="Stretch" />
+          <Setter Property="Padding" Value="12,12,24,12" />
+        </Style>
+      </ListView.ItemContainerStyle>
+    </ListView>
+    <StackPanel x:Name="shapesPanel"
```

Comment:
I think my main comment is that it wasn't super obvious that this was tied to the code-behind shown in the code panel and what the effect was (as it's pretty seamless outside of the show animation on ListView).

Maybe add a comment here alluding to the helper behind the scenes? Also it could be interesting if you offset the icons (without the backgrounds) ***between*** the ListView items, as then that could show a neat use case that'd be hard to do otherwise without these extensions.

Reply:
Added more docs with a XAML comment, and tweaked the effect in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3220/commits/50b455b1822ac7eb8acb19c5b770942b6465b5c4.